src/test/java/test/LoginTests.java

```diff
@@ -1,10 +1,9 @@
 package test;
 
-import PageFactory23.HomePage;
-import PageFactory23.LoginPage;
+import POM.HomePage;
+import POM.LoginPage;
```

Comment:
**Please, remove code redundancies!**

**Don't repeat yourself! ( In Java, it means don't write the same code repeatedly. )**

For example 

```
    @Test (enabled = true)
    public void LoginValidEmailPasswordTest () {
        LoginPage loginPage = new LoginPage(getDriver());
        HomePage homePage = new HomePage(getDriver());
        loginPage.login();
        Assert.assertTrue(homePage.isUserAvatarDisplayed());
    }
```

![image](https://user-images.githubusercontent.com/104761482/207944667-25630baf-193f-4307-8007-0870eab3de17.png)